### PR TITLE
[14.0][MIG] account_invoice_refund_link: Migration to 14.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa 
TT28082

ping @pedrobaeza @ernestotejeda @victoralmau

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

File: account_invoice_refund_link/README.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to "Invoicing -> customers -> Invoices", create an Invoice
+   and validate it by clicking on the 'Post' button.
```
Comment (ernestotejeda) [inline account_invoice_refund_link/README.rst:43]:
```suggestion
   and validate it by clicking on the 'Confirm' button.
```

**Review by ernestotejeda — COMMENTED** [review]
Code and functional review: :+1:

File: account_invoice_refund_link/README.rst (hunk 1 above)
Comment (joao-p-marques) [reply account_invoice_refund_link/README.rst:43]: Sorry, I changed it on `USAGE.rst` but forgot to regenerate the `README.rst`. I think Ocabot does that also on merge, right?

**Review by ernestotejeda — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**